packages/health-check/cli/enable

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env node
+
+process.env.NODE_ENV = process.env.NODE_ENV || "production";
```

Comment:
why?

Reply:
Because otherwise it doesn't make sense, server will only persist the setting if it's `production` environment - check `db.js`, otherwise it's in-memory (for development) and since the memory is not shared, it will not affect the running development process otherwise.